lib/grpc_reflection/service/builder.ex

```diff
@@ -188,7 +269,7 @@ defmodule GrpcReflection.Service.Builder do
     []
   end
 
-  defp package_from_name(service_name) do
+  def package_from_name(service_name) do
```

Comment:
Why is this now a public function?

Reply:
Temporarily changed `defp` to `def` for debugging, but forgot to revert before pushing the branch.

Reply:
If it was meaningful to have access once, it may be again.  This can be made into a `def` in the Builder Util module and then covered by unit tests